src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,18 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine excavator = new Excavator();
+        Machine bulldozer = new Bulldozer();
+        Machine truck = new Truck();
+        Machine[] machines = new Machine[]{excavator, bulldozer, truck};
```

Comment:
variables should be introduced when
a) it's reused in several places
b) we have clearer code by giving understandable name to complicated code part after =

I can't see such situations here
```suggestion
        Machine[] machines = new Machine[]{new Excavator(), new Bulldozer(), new Truck()};
```